backend/schema/message.py

```diff
@@ -29,7 +29,7 @@ class MessageRequest(BaseModel):
     richUiData: Any = {}
     files: List[AnyHttpUrl] = []
     saved_by: List[str] = []
-    timestamp: int = int(datetime.utcnow().timestamp())
+    timestamp: int
```

Comment:
Why didn't you keep the previous default value?

Reply:
Hi @zxenonx That was what the @t_a specified. He requested it be left as an integer instead of the default timestamp I set initially. 

It is also a required field so that it throws an error from the frontend if there's any instance that the data is sent without a timestamp

Reply:
Understood @pauline-banye. Thanks.